decidim-assemblies/app/commands/decidim/assemblies/admin/update_assembly.rb

```diff
@@ -40,7 +40,12 @@ def call
 
         def update_assembly
           @assembly.assign_attributes(attributes)
-          @assembly.save! if @assembly.valid?
+          if @assembly.valid?
```

Comment:
Shouldn't this be wrapped in a transaction? Otherwise the assembly could be created without the log being updated.

Maybe it could be put inside the `perform_action` block so it has an implicit transaction? It could be a good pattern to future-proof this.

Reply:
I wrapped the save and log creation. The `valid?` call does not need to be inside the transaction